category/travel.html

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
Wenn du diese Datei unter `category/travel/index.html` ablegst, bekommst du schöner aussehende URLs. 

Jetzt is die URL: `eva-ernst.me/category/travel.html`

Mit meinem Vorschlag von oben würdest du die URL `eva-ernst.me/category/travel/` erhalten.